commands/browsing/default-browser-chrome.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Dependency: requires defaultbrowser (https://github.com/kerma/defaultbrowser/)
+# Install via Homebrew: `brew install defaultbrowser`
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Switch Default Browser to Chrome
+# @raycast.mode silent
+# @raycast.packageName Browsing
+
+# Optional parameters:
+# @raycast.icon 🧭
+
+# Documentation:
+# @raycast.author Marc Klingen
+# @raycast.authorURL https://github.com/marcklingen
+# @raycast.description Set Chrome as the default browser.
+
+defaultbrowser chrome
```

Comment:
@marcklingen based on this guideline’s section “[Scripts that require installation of runtimes and dependencies](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md#scripts-that-require-installation-of-runtimes-and-dependencies)”, would you please add the validation for the `defaultbrowser` in all scripts?

```suggestion
if ! command -v defaultbrowser &> /dev/null; then
    echo "defaultbrowser command is required (https://github.com/kerma/defaultbrowser).";
    exit 1;
fi 

defaultbrowser chrome
```

Reply:
Looking into the dependency it doesn't do much. Might be easy to build the same in Swift as a script command. Not a requirement though.

https://github.com/kerma/defaultbrowser/blob/master/src/main.m

Reply:
Thank you @unnamedd and @thomaspaulmann for your feedback.

I added the validation (sorry for that) and moved the scripts to the `system` directory. I had a look at the dependency as well as it is rather simple. Unfortunately, I have no experience with Swift to replace it. Maybe this can be done by someone else if the script is useful for others.